src/plugins/data_source_management/public/components/edit_data_source/components/edit_form/edit_data_source_form.tsx

```diff
@@ -804,7 +819,7 @@ export class EditDataSourceForm extends React.Component<EditDataSourceProps, Edi
             value={this.state.auth.credentials?.region || ''}
             onChange={this.onChangeRegion}
             onBlur={this.validateRegion}
-            data-test-subj="updateDataSourceFormRegionField"
+            data-test-subj="editDataSourceFormRegionField"
```

Comment:
Do you need to modify the file "edit_data_source_form.test.tsx" ? Thanks

Reply:
"data-test-subj" attributes are for cypress tests only. I have a PR open in the cypress test repo using those updated values. https://github.com/opensearch-project/opensearch-dashboards-functional-test/issues/665